resources/startup.sh

```diff
@@ -245,48 +209,46 @@ function subsequentSonarStart() {
   sql "UPDATE properties SET text_value='https://${FQDN}/sonar' WHERE prop_key='sonar.core.serverBaseURL';"
```

Comment:
We really need the sql Command here? Can this be done with doguctl template? Or we could save the admin password and use something like set_property_with_default_admin_credentials "sonar.core.serverBaseURL" "https://${FQDN}/sonar"
like before. This way we can remove the whole sql configuration from our script

Reply:
Unfortunately not, these settings are not accepted by SonarQube if they are set in the sonar.properties file. And we do not want to have a persistent admin user.

Reply:
EDIT: This configuration is now set via the REST API.